### PR TITLE
fix: damping error

### PR DESCRIPTION
**Pull request by swapanil**
throwing dumping error in example

### CODE REVIEW

**Review by Simek — COMMENTED** [review]
Hello @swapanil, thank for the update! 👍 

I have two question about this PR:
- what is its relation with #2959, does it supersede it?
- can you provide a detailed description about error you are getting? 

Also, please stick to the common formatting in the examples, by that I mean double quotes and no trailing commas.

**Comment by HyunseungLee-Travis** [comment]
@Simek, it seems like a bug in React Native.
Using `create()` does not work, using objects work.
I created a [new issue](https://github.com/facebook/react-native/issues/33740) in the React Native Repo

File: docs/layoutanimation.md [hunk 1]
```diff
@@ -224,11 +223,11 @@ class App extends Component {
...
         500,
         LayoutAnimation.Types.spring,
-        LayoutAnimation.Properties.scaleXY
-      )
+        LayoutAnimation.Properties.scaleXY,
+      ),
     );
     this.setState({
-      boxPosition: this.state.boxPosition === "left" ? "right" : "left"
+      boxPosition: this.state.boxPosition === 'left' ? 'right' : 'left',
```
Comment (Simek) [inline docs/layoutanimation.md:230]:
```suggestion
      boxPosition: this.state.boxPosition === "left" ? "right" : "left",
```

File: docs/layoutanimation.md [hunk 2]
```diff
@@ -224,11 +223,11 @@ class App extends Component {
       LayoutAnimation.create(
         500,
         LayoutAnimation.Types.spring,
-        LayoutAnimation.Properties.scaleXY
-      )
+        LayoutAnimation.Properties.scaleXY,
+      ),
```
Comment (Simek) [inline docs/layoutanimation.md:227]:
```suggestion
        LayoutAnimation.Properties.scaleXY
      )
```

File: docs/layoutanimation.md [hunk 3]
```diff
@@ -228,7 +227,7 @@ class App extends Component {
       )
     );
     this.setState({
-      boxPosition: this.state.boxPosition === "left" ? "right" : "left"
+      boxPosition: this.state.boxPosition === "left" ? "right" : "left",
```
Comment (Simek) [inline docs/layoutanimation.md:230]:
```suggestion
      boxPosition: this.state.boxPosition === "left" ? "right" : "left"
```

**Comment by Simek** [comment]
Closing as duplicate of #2959.

**Pull request closed without merging**